<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDlg</class>
 <widget class="QDialog" name="ConnectDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <item>
    <widget class="QPushButton" name="btnObserve">
     <property name="text">
      <string>Поиск устройств</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/res/find.png</normaloff>:/icons/res/find.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Найденные устройства</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cmbDevice"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnConnect">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Соединиться</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/res/connect.png</normaloff>:/icons/res/connect.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkAutoConnect">
     <property name="text">
      <string>Подключаться автоматически</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkScanOnStart">
     <property name="text">
      <string>Автозапуск сканирования</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkAutoLoadCfg">
     <property name="text">
      <string>Загрузка с устройства после соединения</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbState">
     <property name="text">
      <string>Ждём команду</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
